metadata-ingestion/tests/integration/kafka-connect/test_kafka_connect_s3sink.py

```diff
@@ -0,0 +1,105 @@
+import time
```

Comment:
I'd prefer to have this test ported in `test_kafka_connect.py` similar to `test_kafka_connect_mongosourceconnect_ingest` as that would start the kafka connect docker based setup only once for all tests. When that's done, you'd need to add `connector_patterns` config in [kafka_connect_s3sink_to_file.yml](https://github.com/datahub-project/datahub/pull/8298/files#diff-90fe7c3cba87f500f7bc3fc5664ce8b91c98d00bf17c9a01e0b8fb29c3b30853) to include only s3 connector related mcps.

Could you pull images before running the test and confirm if it succeeds on your end this time ?

`docker compose -f tests/integration/kafka/docker-compose.yml -f tests/integration/kafka-connect/docker-compose.override.yml pull`

Reply:
Cool, let me give it a try.

Reply:
I still having timeout issues with the tests - tried increasing the timeout values but didn’t help either. I’ll see if the tests run here.

Reply:
@mayurinehate pls see the errors on my fork: https://github.com/tusharm/datahub/actions/runs/5517230132. Any suggestion?

Reply:
Hey @tusharm, did you run this first - docker compose -f tests/integration/kafka/docker-compose.yml -f tests/integration/kafka-connect/docker-compose.override.yml pull ?

I checked and the test should work fine on CI after [this change](https://github.com/datahub-project/datahub/pull/8298#discussion_r1259519617).

Reply:
Thanks, fixed now.